src/style.css

```diff
@@ -1,3 +1,59 @@
-body {
+  body {
+  font-family: Roboto, sans-serif;
   margin: 0;
-}
+  font-size: 12px;
+  font-weight: 500;
+  }
+
+  .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  text-transform: uppercase;
+  }
+
+  .logo {
+  width: 40px;
+  height: 40px;
+  }
+
+  .nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  }
+
+  .nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
+  line-height: 60px;
+  list-style: none;
+  }
+
+  .nav__item:not(:first-child) {
+  margin-left: 20px;
+  }
+
+  .nav__link:hover {
+  color: #00ACDC;
+  }
+
+  .is-active {
+  position: relative;
+  color: #00ACDC;
```

Comment:
move all repeatable colors to some vars and reuse them everywhere

Reply:
I did this, but I can't add git push in gitbush, because  because it gives the following error in gitbush:
-$ git add readme.md src/
fatal: Unable to create 'C:/projects/layout_moyo-header/.git/index.lock': 
File exists.

Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue.
- $ git push
fatal: The current branch develop has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin develop

To have this happen automatically for branches without a tracking
upstream, see 'push.autoSetupRemote' in 'git help config'.
-$ npm run deploy

> header@1.0.0 deploy C:\projects\layout_moyo-header
> mate-scripts deploy

Start deploy to gh-pages. Please wait, it may take up to minute.

Deploy error:  Command failed: git add C:\projects\layout_moyo-header\dist

  :root {
  --color-brand: #00ACDC;
  }

  body {
  font-family: Roboto, sans-serif;
  margin: 0;
  font-size: 12px;
  font-weight: 500;
  }

  .header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
  text-transform: uppercase;
  }

  .logo {
  width: 40px;
  height: 40px;
  }

  .nav__list {
  display: flex;
  margin: 0;
  padding: 0;
  list-style: none;
  }

  .nav__link {
  display: block;
  text-decoration: none;
  color: black;
  line-height: 60px;
  list-style: none;
  }

  .nav__item:not(:first-child) {
  margin-left: 20px;
  }

  .nav__link:hover {
  color: #00ACDC;
  color: var(--color-brand);
  }

  .is-active {
  position: relative;
  color: var(--color-brand);
  }

  .is-active::after {
  position:absolute;
  top: 56px;
  left: 0;
  bottom: 0;
  border-radius: 8px;
  content:'';
  width: 37px;
  height: 4px;
  background-color: var(--color-brand);
  }


-